src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -145,6 +149,7 @@ public void close() throws IOException {
     }
 
     public SeekableByteChannelPrefetcher(SeekableByteChannel chan, int bufSize) throws IOException {
+        this.prefetcherIndex = (prefetcherCount++);
```

Comment:
As long as nothing will be broken by potentially having two threads with identical names than I'm fine letting through the rare case of having a race condition in our thread naming...